terraform/shared/elastic.tf

```diff
@@ -1,3 +1,7 @@
+data "ec_deployment" "logging" {
+  id = "f4c9aca09347dad4c05f35cfbec04cdb"
+}
```

Comment:
FYI, the logging cluster is defined in Terraform, so you could pick up this ID from that state: https://github.com/wellcomecollection/platform-infrastructure/blob/main/critical/elastic_logging_cluster.tf